addons/exact-join-date/script.js

```diff
@@ -0,0 +1,8 @@
+var join_date = `${document.querySelector(".profile-details span:nth-child(2)").title}`;
+let join_date_parts = join_date.split("-");
+let months = ["January", "Febuary", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+var join_date_improved = months[Number(join_date_parts[1]) - 1] + " " + Number(join_date_parts[2]) + ", " + Number(join_date_parts[0]);
+let details = document.querySelector(".profile-details");
+details.innerHTML = details.innerHTML.replace("ago", "");
+document.querySelector(".profile-details span:nth-child(2)").title = document.querySelector(".profile-details span:nth-child(2)").innerHTML.replace(/&nbsp;/g, " ") + " ago";
+document.querySelector(".profile-details span:nth-child(2)").innerHTML = join_date_improved;
```

Comment:
Setting `innerHTML` can be a security risk. I don't know if it would lead to a vulnerability here, but it's better to set `textContent` where possible. Learn more [here](https://scratchaddons.com/docs/develop/userscripts/best-practices/#avoid-using-innerhtml).

Reply:
It's completely remaking the entire `.profile-details` section using innerHTML. I'm not entirely sure right now, but there's a chance this breaks elements such as the follow button or the upload profile picture function.